tests/functional/ui/test_cephfs_subvolume_metrics.py

```
@@ -463,3 +469,324 @@ def test_cephfs_subvolume_top_10_ranking(self, metric, threading_lock):
+                rate="100m",
+                runtime=900,
+            )
+            multi_pods.append(pod_obj)
+
+        request.cls.single_project = single_project
+        request.cls.multi_project = multi_project
+        request.cls.multi_pods = multi_pods
+
+        all_projects = [single_project, multi_project]
+
+        def finalizer():
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `finalizer`

Add return type annotation: `None`

(ANN202)